apix-impl/src/integration-test/java/eu/xenit/apix/tests/search/SearchServiceTest.java

```diff
@@ -438,4 +438,30 @@ public void invalidLong_onlyNode_doesNotThrowException() {
         searchService.query(searchQuery);
     }
 
+    @Test
+    public void testSearchForQuotationMarkProperty() {
+        final String testString = "For testing \"quotes\".";
+
+        transactionService.getRetryingTransactionHelper().doInTransaction(() -> {
```

Comment:
Can you not use the `baseTest.init()` method to set this up, at least partially?